app/frontend/src/features/donations/constants.ts

```diff
@@ -0,0 +1,50 @@
+export const DONATIONS_BANNER_TITLE =
+  "Your help is needed to support our mission";
+export const DONATIONS_BANNER_TEXT =
+  "Your donation will support the future of couchsurfing";
+export const DONATIONS_IMAGE_ALT = "Couchers banner" 
+export const DONATIONS_TITLE = "You can build the future of couchsurfing";
+export const DONATIONS_TEXT = `[Couchers.org](https://couchers.org) is a non-profit that is run on donations, similar to the model used by Wikipedia.
+
+We believe that there is enough good will in the couch surfing community to build a platform that will easily outcompete Couchsurfing, and restore ownership to all of us.
+
+Couchsurfing was built by the community, and we fully intend to put it back in their hands by building an amazing product. We need your help to do so.
+
+[Couchers.org](https://couchers.org) is a place to connect with others, not a place to run advertising.
```

Comment:
There are only 2 actual links on the desktop mockups, having too many looks too busy imo

Reply:
Here I just copy pasted the text from figma, let me know if you want to modify from that.